components/BasketPage.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from 'react';
+import {
+  SafeAreaView,
+  View,
+  FlatList,
+  StyleSheet,
+  Text,
+  StatusBar,
+  Image,
+  Pressable,
+} from 'react-native';
+
+const BasketPage = ({ basket }) => {
+    const tit = "empty";
+
+    if (basket) {
+        tit = basket[0]?.title;
```

Comment:
does tit = title?

Reply:
yes, I just tried to check if basked have been passed